test/run-tests.sh

```diff
@@ -312,19 +319,24 @@ examples() {
     (cd "$scriptDir/../examples" && nix-shell --run "$script")
 }
 
+shellcheck() {
+  . "$scriptDir/ci/shellcheck.sh"
+}
```

Comment:
nit: Doesn't it make more sense to have shellcheck.sh in the `tests/` directory?